src/support/ResourcePolicies.tsx

```diff
@@ -0,0 +1,66 @@
+import ReactJson from '@microlink/react-json-view';
+import React, { useEffect, useState } from 'react';
+import { Ajax } from 'src/libs/ajax';
+import colors from 'src/libs/colors';
+import { reportError } from 'src/libs/error';
+import { ResourceTypeSummaryProps } from 'src/support/SupportResourceType';
+
+export const ResourcePolicies = (props: ResourceTypeSummaryProps) => {
+  const [resourcePolicies, setResourcePolicies] = useState<object>();
+  const [errorMessage, setErrorMessage] = useState<string>('');
+
+  function clear() {
+    setErrorMessage('');
+    setResourcePolicies(undefined);
+  }
+
+  useEffect(() => {
+    const loadResourcePolicies = async () => {
+      clear();
+      if (props.fqResourceId.resourceId) {
+        try {
+          const policies = await Ajax().SamResources.getResourcePolicies(props.fqResourceId);
+          policies instanceof Array && policies.length === 0
```

Comment:
Generally, suggest `Array.isArray` vs `instanceof Array`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray#description

```suggestion
          Array.isArray(policies) && policies.length === 0
```